circleci/test-eof-newline

```diff
@@ -7,7 +7,7 @@ exit_code = 0
 FNULL = open(os.devnull, 'w')
 
 for filename in subprocess.check_output(["git", "ls-files"]).split("\n"):
-    if os.path.isfile(filename) and not filename.startswith('.idea/'):
+    if os.path.isfile(filename) and not filename.startswith('.idea/') and not filename.endswith('.icc'):
```

Comment:
This is wrong, in place of that you should add `*.icc binary` in the `.gitattributes`...

Reply:
Changed it.

Reply:
Thanks :-)